src/linalg.jl

```diff
@@ -297,8 +297,8 @@ end
     end
 end
 
-# TODO same for `RowVector`?
 @inline Size(::Union{RowVector{T, SA}, Type{RowVector{T, SA}}}) where {T, SA <: StaticArray} = Size(1, Size(SA)[1])
+@inline Size(::Union{RowVector{T, CA}, Type{RowVector{T, CA}}} where CA <: ConjVector{<:Any, SA}) where {T, SA <: StaticArray} = Size(1, Size(SA)[1])
```

Comment:
Oops. This line of code was meant for #157. I suggest we review all changes here and just merge both (or this).